src/main/java/frc/utils/beamBreaker/digitalbeambreaker/DigitalBeamBreaker.java

```diff
@@ -0,0 +1,24 @@
+package frc.utils.beambreaker.digitalbeambreaker;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.utils.beambreaker.BeamBreakerInputsAutoLogged;
+import frc.utils.beambreaker.IBeamBreaker;
+
+public class DigitalBeamBreaker implements IBeamBreaker {
+
+	private final DigitalInput beambreaker;
+
+	private final Debouncer debouncer;
+
+	public DigitalBeamBreaker(int channel, double debounceTime, Debouncer.DebounceType debounceType) {
```

Comment:
Add a param for normally open and normally closed. (if you do not understand feel free to contact me)